libs/functions/generate_api.py

```diff
@@ -84,73 +86,50 @@ def _generate_api_for_projections(projection_row):
-        cumulativeNegativeTests=_get_or_none(
-            projection_row[rc.CUMULATIVE_NEGATIVE_TESTS]
-        ),
+        population=actual_data.get(CommonFields.POPULATION),
+        intervention=intervention.name,
+        cumulativeConfirmedCases=actual_data[CommonFields.CASES],
+        cumulativeDeaths=actual_data[CommonFields.DEATHS],
+        cumulativePositiveTests=actual_data.get(CommonFields.POSITIVE_TESTS),
+        cumulativeNegativeTests=actual_data.get(CommonFields.NEGATIVE_TESTS),
         hospitalBeds={
-            "capacity": projection_row[rc.PEAK_BED_CAPACITY],
-            # TODO(chris): Get from assembled sources about current hospitalization data.
-            # i.e. NV data we can manually update.
-            "currentUsage": None,
-            "typicalUsageRate": state_beds_data[BedsDataset.Fields.ALL_BED_TYPICAL_OCCUPANCY_RATE],
+            "capacity": capacity,
+            "totalCapacity": total_bed_capacity,
+            "currentUsageCovid": actual_data.get(CommonFields.CURRENT_HOSPITALIZED),
+            "typicalUsageRate": typical_usage_rate,
         },
         ICUBeds={
-            # Note(Chris): We do not currently pass through ICU Bed capacity calculations
-            # in the projection_row.  This wouldn't be a ton of work to do, but
-            # using the provided beds data for the time being.
-            "capacity": state_beds_data[BedsDataset.Fields.ICU_BEDS],
-            "currentUsage": None,
-            "typicalUsageRate": state_beds_data[BedsDataset.Fields.ICU_TYPICAL_OCCUPANCY_RATE],
+            "capacity": actual_data.get(CommonFields.ICU_BEDS),
+            "totalCapacity": actual_data.get(CommonFields.ICU_BEDS),
+            "currentUsageCovid": actual_data.get(CommonFields.CURRENT_ICU),
+            "typicalUsageRate": actual_data.get(CommonFields.ICU_TYPICAL_OCCUPANCY_RATE),
         },
     )
 
 
-def _generate_county_actuals(projection_row, state_intervention, county_beds_data):
-    intervention_str = state_intervention.name
-
-
-    icu_beds = None
-    if county_beds_data:
-        icu_beds = {
-            "capacity": county_beds_data[BedsDataset.Fields.ICU_BEDS],
-            "currentUsage": None,
-            "typicalUsageRate": county_beds_data[BedsDataset.Fields.ICU_TYPICAL_OCCUPANCY_RATE],
-        }
-
-    return _Actuals(
-        population=projection_row[rc.POPULATION],
-        intervention=intervention_str,
-        cumulativeConfirmedCases=projection_row[rc.CURRENT_CONFIRMED],
-        cumulativeDeaths=projection_row[rc.CURRENT_DEATHS],
-        cumulativePositiveTests=None,
-        cumulativeNegativeTests=None,
-        hospitalBeds={
-            "capacity": projection_row[rc.PEAK_BED_CAPACITY],
-            "currentUsage": None,
-            "typicalUsageRate": county_beds_data.get(BedsDataset.Fields.ALL_BED_TYPICAL_OCCUPANCY_RATE),
-        },
-        ICUBeds=icu_beds,
-    )
+def _generate_actuals_timeseries(actuals_timeseries_dataset, intervention):
```

Comment:
i'm ok with this, since it's how it was done before, but i'd probably like move away from an iter-row based data transformation approach.